src/libraries/Errors.sol

```diff
@@ -30,6 +30,9 @@ library Errors {
     /// @notice Thrown when trusted forwarder can not execute the actions.
     error NotPossible();
```

Comment:
```suggestion
    error IncorrectActionCount();
```

Reply:
done in https://github.com/aragon/staged-proposal-processor-plugin/pull/8/commits/9cb984c4cbe559a051943fde540da25b853c2400